browser-sign-in/.expo/README.md

```diff
@@ -0,0 +1,8 @@
+> Why do I have a folder named ".expo" in my project?
+The ".expo" folder is created when an Expo project is started using "expo start" command.
```

Comment:
Is this directory something we'd like to add to the `.gitignore` file?

Reply:
Yes you're right. I will add this to gitignore